model-engine/model_engine_server/inference/vllm/build_and_upload_image.sh

```diff
@@ -1,15 +1,21 @@
 #!/bin/bash
 
-# Build and push vLLM docker image to ECR.
+# Build and push vLLM docker image to AWS ECR.
 
 set -eo pipefail
 
 if [ -z "$1" ]; then
+  echo "Must supply AWS account ID"
+  exit 1;
+fi
+
+if [ -z "$2" ]; then
   echo "Must supply the image tag"
   exit 1;
 fi
 
-IMAGE_TAG=$1
-aws ecr get-login-password --region us-west-2 | docker login --username AWS --password-stdin 692474966980.dkr.ecr.us-west-2.amazonaws.com
-DOCKER_BUILDKIT=1 docker build -t 692474966980.dkr.ecr.us-west-2.amazonaws.com/vllm:$IMAGE_TAG .
-docker push 692474966980.dkr.ecr.us-west-2.amazonaws.com/vllm:$IMAGE_TAG
+IMAGE_TAG=$2
+ACCOUNT=$1
+aws ecr get-login-password --region us-west-2 | docker login --username AWS --password-stdin $ACCOUNT.dkr.ecr.us-west-2.amazonaws.com
```

Comment:
Might want to pass in a full ECR URL instead of just the account, because this leaks info about the region. But think this is ok for now.